src/openrct2/ride/CableLift.cpp

```diff
@@ -115,10 +115,10 @@ void Vehicle::CableLiftUpdate()
  */
 void Vehicle::CableLiftUpdateMovingToEndOfStation()
 {
-    if (velocity >= -439800)
+    if (velocity >= -6.710816_MPH)
```

Comment:
I’m reeeeeeally not sure if I like this. It feels finicky and prone to failure to me.

Reply:
And `kupt` is supposed to mean `kilo units per tick`?

I don't know, if it makes all numbers smaller, easier to read, seems like a good change, but then you also introduce a unit name that most people are unaware of?

Reply:
There was a suggestions that the conversion is 2^15 and not 2^16 that locomotion uses. the way to tell would be comparisons between mph16 and mph32

Reply:
Not sure I understand how this changes things

Reply:
It would mean multiplying all mph32 values by 2 and using 32768 as the conversion instead of 65536

Reply:
Ahh seems its none of the above: mph32 to mph16 is: `(vehicle->velocity * 9) >> 18;` i.e. 2.25 / 65536. So we need to do something like  `speedMph * 147456` so this user would become `2.9825_mph`.